scripts/hack/patch-rancher-logging

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+patch_rancher_logging_chart()
+{
+  local chart_dir=$1 #${CHARTS_DIR}
+  local logging_version=$2 #LOGGING_VERSION
+  local pkg_logging_path=$3 #${PKG_PATCH_LOGGING_PATH}
+  local cwd=$(pwd)
+
+  if [ ! -d "${pkg_logging_path}/${logging_version}" ]; then
+    echo "NOTE: there is no related path: ${pkg_logging_path}/${logging_version} to patch, SKIP"
+    return 0
+  fi
+
+  cd ${chart_dir}
+  tar zxf rancher-logging-${logging_version}.tgz
+
+  echo "patch rancher logging rke2 configmap.yaml, daemonset.yaml"
+  cp -f ${pkg_logging_path}/${logging_version}/configmap.yaml ./rancher-logging/templates/loggings/rke2/configmap.yaml
+  cp -f ${pkg_logging_path}/${logging_version}/daemonset.yaml ./rancher-logging/templates/loggings/rke2/daemonset.yaml
+  grep "harvester-patch" ./rancher-logging/templates/loggings/rke2/configmap.yaml || true
```

Comment:
Just curious, what's the use of these two lines? Are they debugging code? Thanks!

Reply:
Yes, for debug, we add `#harvester-patch` in `configmap.yaml` and `daemonset.yaml`, for those lines we added;  here it gives us a quick hint that the files are trully replaced.